packages/admin/admin/src/index.ts

```diff
@@ -22,6 +22,7 @@ export { SplitButtonContext, SplitButtonContextOptions } from "./common/buttons/
 export { useSplitButtonContext } from "./common/buttons/split/useSplitButtonContext";
 export { ClearInputAdornment, ClearInputAdornmentProps } from "./common/ClearInputAdornment";
 export { CometLogo } from "./common/CometLogo";
+export { Dialog, DialogClassKey, DialogProps } from "./common/Dialog";
```

Comment:
We should probably add an import restriction to prevent using MUIs `Dialog`. 
This is how it's done for `Alert` for example: 

https://github.com/vivid-planet/comet/blob/f8ae0843c9fc3b12f803b0bf961d898eb11e5ac5/packages/eslint-config/react.js#L42-L46

Reply:
The import restriction is in another PR, since it should be activated in next: #2610.